subcommand/injector/command.go

```diff
@@ -296,18 +307,17 @@ func (c *Command) certWatcher(ctx context.Context, ch <-chan cert.Bundle, client
 
 		crt, err := tls.X509KeyPair(bundle.Cert, bundle.Key)
 		if err != nil {
-			log.Error(fmt.Sprintf("Error loading TLS keypair: %s", err))
+			log.Warn(fmt.Sprintf("Could not load TLS keypair: %s. Trying again...", err))
 			continue
 		}
 
 		isLeader := true
 		if c.flagUseLeaderElector {
 			// Only the leader should do the caBundle patching in k8s API
 			var err error
-			le := leader.New()
-			isLeader, err = le.IsLeader()
+			isLeader, err = leaderElector.IsLeader()
 			if err != nil {
-				log.Error(fmt.Sprintf("error checking leader: %s", err))
+				log.Warn(fmt.Sprintf("Could not check leader: %s. Trying again...", err))
```

Comment:
I noticed that some log lines are capitalized, and in other cases not. I guess we should probably be consistent either way.

Reply:
Sure, I think generally log messages should start with a capitalized word, and errors themselves should start with lowercase.

Reply:
Totally agree 👍